tests/AppendStreamTest.php

```diff
@@ -198,8 +198,9 @@ public function testCatchesExceptionsWhenCastingToString()
         self::assertFalse($a->eof());
 
         $errors = [];
-        set_error_handler(function (int $errorNumber, string $errorMessage) use (&$errors) {
+        set_error_handler(static function (int $errorNumber, string $errorMessage) use (&$errors): bool {
             $errors[] = ['number' => $errorNumber, 'message' => $errorMessage];
+            return false;
```

Comment:
why false? that makes the test fail

Reply:
before this function was returning `void` in other words `null`

so my thinking was this: https://3v4l.org/gWGh8

Is there a chance the tests could be wrong.. or I am the one missing something?

Reply:
You cannot return false as it means calling forwarding to the phpunit error handler which makes the test fail.

Reply:
done.